### ISSUE TITLE
Hack's derivates in Readme

### ISSUE THREAD

**Issue by dok42** [source-foundry/Hack#355]
Hello
Would be possible to add somewhere in Readme, a section for Hack derivates ?
Just to let other people to easily find those.

There is Alt-Hack for those who want to customize themself.

**Comment by chrissimpkins** [comment]
Definitely willing to support this.  Shall we leave this thread open for a bit to collect information on what is currently available out there?

**Comment by dok42** [comment]
Sure, I hope more people use this really good monospace font,

**Comment by vl4dimir** [comment]
I like the initiative! 😃  I've created a mod with a slashed zero and slab "i", you can find it [here](https://github.com/BRUTALISM/Hack).

I appreciate all the tiny tweaks and improvements that have been done to Hack over time, but I've always found the curved "i" and the diamond zero a bit distracting. I generally like these in "Menlo style" but Menlo's butchering of uppercase M and N prevent me from using it. ¯\_(ツ)_/¯

Let me know if you want me to do some tweaks to my repo in terms of README style etc. I'd like to keep it up to date with this repo as much as I can.

**Comment by chrissimpkins** [comment]
> Let me know if you want me to do some tweaks to my repo in terms of README style etc. I'd like to keep it up to date with this repo as much as I can.

None necessary from our standpoint.  Feel free to describe your derivative however suits your needs.  It is not necessary to maintain the upstream README text there if it does not help your users.  Let us know how we can facilitate updates downstream as we make changes here.

Thanks for sharing this with us!

**Comment by chrissimpkins** [comment]
@dok42 will you please add the link to your derivative to this thread so that users can find it and so that we maintain it once we are prepared to document available derivatives that are out there?

**Comment by tflo** [comment]
> I've always found the curved "i" […] a bit distracting

LOL, the “i” is one of the main reasons why I like this font :) Hack is one of the few fonts that have a curved “i”, and – such a nice one!

From time to time I switch to another, more “squary”, font (Plex, San Francisco, …), just to entertain my eyes with something different. But each time I come back to Hack it’s a pleasure to see that lovely “i”. Guess, I’m in love with Hack’s “i” :)

– Tom

**Comment by chrissimpkins** [comment]
@tflo Thanks for the very kind comments!

If you like the `i` but happen to dislike other glyph designs, the alt-hack repository is being built with a number of alternate designs so that you can roll your own version in an a la carte fashion...

https://github.com/source-foundry/alt-hack

Alt-hack is open to contributions from anyone who has new alternate glyph design ideas.  Support for custom versions that you can put out there for others was very much the impetus for this new library.   Build it and push it to a repo then post a link here so that we can share it with others who might have the same subjective inclinations :)

**Comment by chrissimpkins** [comment]
For those who currently have derivatives and others who may wander across this thread, I'll also mention that this script supports font renaming so that you can install builds side-by-side with Hack (and any number of other custom builds) if you want to modify the name of any derivative build:

https://github.com/chrissimpkins/fontname.py

At some stage, we will push something more formal as an executable application, but the above script does the job for now...

**Comment by tflo** [comment]
> If you like the i but happen to dislike other glyph designs, the alt-hack repository is being built with a number of alternate designs so that you can roll your own version

Yeah, I guess I know about that. But, you know, I got a bug in my brain. It’s called perfectionism.
One of the effects of that bug is that I tend to spend days and weeks just to _configure_ something to what I think it should be. Even without too much imagination you can imagine that such actions often end in a loop ;)

I discovered that bug many years ago, and for some years now I’m trying to avoid it: The workaround is: Take the stuff as it comes out of the box — if it is _halfway_ acceptable.

This works fine, more or less. From time to time I’m experiencing relapses, but it’s fine. I think.

But, please: 

do.not.induce.me.to.configure.that.damn.font.to.what.I.think.it.should.be!

This would be the end of my productivity ;)

Don’t get me wrong: I’m not saying there is big stuff to improve! But… argh ;)


– Tom

**Comment by chrissimpkins** [comment]
@tflo hahahaha! Come to the dark side.

**Comment by tflo** [comment]
😷

**Comment by dok42** [comment]
@chrissimpkins Just did read it now.
Sure, here is the [link](https://github.com/dok42/Hack-dok42).
And [here](https://github.com/dok42/HackD-windows-installer/releases) is the Windows Installer.

**Comment by chrissimpkins** [comment]
@dok42 thank you!

**Comment by chrissimpkins** [comment]
Something that I would still like to do.  If anyone knows of other derivatives out there, please list them here.  There are packages in Nerd Fonts, the Powerline patch repository, and some other areas that I am aware of.  I really like this idea.  Given the length of our README, this will likely end up in a different document with a link from the README.
